src/Patch/Element.js

```diff
@@ -228,6 +228,35 @@ export default function(internals) {
     console.warn('Custom Elements: `Element#insertAdjacentElement` was not patched.');
   }
 
+  function patch_insertAdjacentHTML(destination, baseMethod) {
+    Utilities.setPropertyUnchecked(destination, 'insertAdjacentHTML',
+      /**
+       * @this {Element}
+       * @param {string} where
+       * @param {string} html
+       */
+      function(where, html) {
+        baseMethod.call(this, where, html);
+
```

Comment:
Could you add `where = where.toLowerCase();` here, before the comparison below? [This string doesn't have to match case-sensitively.](https://w3c.github.io/DOM-Parsing/#dom-element-insertadjacenthtml)